media/static-bundles.json

```diff
@@ -102,6 +102,12 @@
       ],
       "name": "grants"
     },
+    {
```

Comment:
Instead of folding the JS and CSS into each page's  static bundle, I made individual JS and CSS bundles to be tacked on a template level. 

This execution was chosen because some of the pages do not have bundles defined. 

❗ An alternate approach is to use the stand-alone bundles only when there are no other bundles available for a specific template.

Reply:
I think this is good for the JS - with HTTP2 this might even be preferable. It'd be nice if we could get a `defer` on that script tag... that can be a V2 thing.

With the CSS include going in the header though it becomes render-blocking with all the potential problems of an extra request. I think it would be better for the CSS to use stand-alone bundles only when there are no other bundles available.

Reply:
Commenting here to say this point is resolved, but wanna leave it up for posterity. I updated the commit to bundle the sticky promo SCSS with the release notes CSS bundle, as it already existed. 👍